packages/ra-ui-materialui/src/detail/Show.js

```diff
@@ -86,6 +86,7 @@ export const ShowView = ({
                             hasList,
                             hasEdit,
                             resource,
+                            ...actions.props
```

Comment:
In here `basePath`, `resource`, `hasList`, `hasEdit`, `data` cannot be customized without it.